be/src/connector/hive_connector.h

```diff
@@ -143,6 +143,9 @@ class HiveDataSource final : public DataSource {
     // iceberg equality delete column slots.
     std::vector<SlotDescriptor*> _equality_delete_slots;
 
+    // iceberg equality delete column tuple desc.
+    TupleDescriptor* _delete_column_tuple_desc;
```

Comment:
TupleDescriptor is deprecated,  suggest to use std::vector<ColumnRef> in BE, List<SlotRef> in FE

Reply:
it seems the type: TupleDescriptor should be passed to variable `tuple_desc` under class `HdfsScannerContext`, and i can't reconstruct HdfsScannerContext without TupleDescriptor in this issue.